### PR TITLE
Planning your project

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

Adam and Andi, I need a quick read and your honest opinions.  I had the idea of someone new to Keptn.  They've been through the introductory material and now are ready to create their first project, possibly for production.  It seemed that we needed a little document that supports them as they plan their first project.

So I scratched this out.  At the very best, it needs more topics and more depth -- maybe info about pros and cons of different approaches or alternate ways of accomplishing a task.  But it may be that this is the sort of piece that writers think are cool and nobody else ever reads.  I keep going back and forth so need other opinions before I do more work.  It may be that these considerations and options should be just folded into the individual pages in the "Define your project" section.  And, right now, that section just has the relevant pieces that are in the existing docs -- virtually no work has been done on them.

If you think a piece like this is worthwhile, feel free to make suggestions about additional material that should be included.  But if this is just a stupid addition, let's say so before I waste more time on it.

### CODE REVIEW

File: content/docs/0.19.x/manage/plan/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+It is important to understand that a Keptn project is **not** a pipeline.
+Instead, activities are triggered by specific events.
+For a broader explanation of this, you can read the seminal
+[Continuous Delivery without pipelines -- How it works and why you need it](https://medium.com/keptn/continuous-delivery-without-pipelines-7e84db8c8261) article.
+
+## What should this project do?
+
+Before you can run the Keptn command to create a project,
+you must populate a [shipyard.yaml](../../reference/files/shipyard) file
+that defines the `stages` for your project.
```
Comment (grabnerandi) [inline content/docs/0.19.x/manage/plan/index.md:24]: defines stages and sequences of tasks that your keptn project will be able to orchestrate

File: content/docs/0.19.x/manage/plan/index.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "dev"
+    - name: "hardening"
+    - name: "production"
+```
+
+In order to do any work, each stage must have at least one sequence and most sequences will contain one or more tasks but this information can be added to the shipyard after you create your project.
```
Comment (grabnerandi) [inline content/docs/0.19.x/manage/plan/index.md:42]: you say here that a stage always needs to have at least one sequence - but - in the example above there are no sequences. I think its still true that there is always the "evaluation" sequence there by default (=implicit) even if you dont specify it. this is why a basic shipyard that only contain stages actually works as every stage by default has a sequence called "evaluation" with a single task that is also called evaluation. A user can however define his own evaluation sequence with any set of tasks

**Review by grabnerandi — COMMENTED** [review]
i like this page and added some comments

File: content/docs/0.19.x/manage/plan/index.md (hunk 2 above)
Comment (agardnerIT) [reply content/docs/0.19.x/manage/plan/index.md:42]: Should we be “advertising” this though? I understand why it is so, but It breaks the premise that the Shipyard is the source of truth and IMO promotes bad habits.

Comment (StackScribe) [reply content/docs/0.19.x/manage/plan/index.md:42]: I stole the statement that "Each stage must have  a sequence" from somewhere in the existing docs and then started to see that this didn't exactly make sense.  Thanks for the explanation, Andi -- now I understand.   For this section, I'm not sure we even need to mention sequences.  But I am going to add this subtle detail to the shipyard reference page.

File: content/docs/0.19.x/manage/plan/index.md (hunk 1 above)
Comment (StackScribe) [reply content/docs/0.19.x/manage/plan/index.md:24]: I did a bit of rewording -- see what you think.

File: content/docs/0.19.x/manage/plan/index.md (hunk 2 above)
Comment (StackScribe) [reply content/docs/0.19.x/manage/plan/index.md:42]: This is way too in-the-weeds for this document but I added info about the implicit evaluation sequence to the shipyard reference file in a separate PR

**Review by grabnerandi — APPROVED** [review]
lgtm

**Review by agardnerIT — APPROVED** [review]
All good from me

**Pull request merged**